### ISSUE TITLE
The pipe 'safe' could not be found!

### ISSUE THREAD

**Issue by crystalYY** [embarq/safe-pipe#3]
hello ,I have read the readme doc, and follow the step, but i got this error 
```
zone-evergreen.js:172 Uncaught Error: The pipe 'safe' could not be found!
    at getPipeDef$1 (core.js:36622)
    at Module.ɵɵpipe (core.js:36585)
    at AgreementDialogComponent_Template (agreement-dialog.component.html:2)
    at executeTemplate (core.js:11937)
    at renderView (core.js:11723)
    at renderComponent (core.js:13239)
    at renderChildComponents (core.js:11526)
    at renderView (core.js:11749)
    at TemplateRef.createEmbeddedView (core.js:15253)
    at ViewContainerRef.createEmbeddedView (core.js:15360)
```
I was use this in agreement-dialog.component, and it is a sub component in shared file
this is my agreement-dialog.component
```ts
url: string;
  signed_agreement:boolean = true;

  constructor(private sanitizer:DomSanitizer, private http: WebApiService) { }

  ngOnInit() {
    this.http.getAgreementById(this.agreementId).subscribe(res => {
      // this.url = this.sanitizer.bypassSecurityTrustResourceUrl(res.data);
      this.url = res.data;
    });
  }
```
and this is my package.json
```json
"dependencies": {
    "@angular/animations": "^9.0.2",
    "@angular/cdk": "^9.1.0",
    "@angular/common": "^9.0.2",
    "@angular/compiler": "^9.0.2",
    "@angular/core": "^9.0.2",
    "@angular/flex-layout": "^9.0.0-beta.29",
    "@angular/forms": "^9.0.2",
    "@angular/localize": "^9.0.3",
    "@angular/platform-browser": "^9.0.2",
    "@angular/platform-browser-dynamic": "^9.0.2",
    "@angular/router": "^9.0.2",
    "@ng-bootstrap/ng-bootstrap": "^6.0.0",
    "angularx-qrcode": "^2.1.0",
    "core-js": "^3.6.4",
    "pdfjs-dist": "^2.0.489",
    "punycode": "^2.1.1",
    "rxjs": "^6.5.4",
    "safe-pipe": "^0.3.0",
    "tslib": "^1.11.0",
    "zone.js": "~0.10.2"
  },
```
thank you

**Comment by embarq** [comment]
@crystalYY if it's still any relevant, I've just published [safe-pipe@1.0.1](https://www.npmjs.com/package/safe-pipe) with full support of [angular@9.x](https://angular.io/)

**Comment by embarq** [comment]
https://github.com/embarq/safe-pipe/commit/e1f2c49522e38bc017bf46294f96c402e1bef5bd
